swarm_test.go

```diff
@@ -396,6 +396,65 @@ func TestLocalStoreAndRetrieve(t *testing.T) {
 	}
 }
 
+// TestTagPersistence tests that a tag is persisted then reloaded when the node is started again
+func TestTagPersistence(t *testing.T) {
```

Comment:
This test should test Swarm.Stop method which contains important part of this PR, beside tags load in NewSwarm.

Reply:
`Stop()` could not be called due to Hive not being started after a call to `NewSwarm`. I added a `started` field to `Hive` in https://github.com/ethersphere/swarm/pull/1894/commits/bbab6acc9ae8af7e21c1955d46d2592899e52f6e so we can still invoke `Stop` regardless